app/javascript/flavours/glitch/features/getting_started_misc/index.jsx

```diff
@@ -68,3 +66,5 @@ class gettingStartedMisc extends ImmutablePureComponent {
   }
 
 }
+
+export default connect()(injectIntl(gettingStartedMisc));
```

Comment:
Can we fix the casing of this class while at it?

Reply:
Sure.